### PR TITLE
add more polish translations

### PR DESCRIPTION
**Pull request by JacekDuszenko**
Add some more polish translations to minikube

### CODE REVIEW

File: translations/pl.json [hunk 1]
```diff
@@ -906,38 +906,38 @@
...
-	"usage: minikube addons configure ADDON_NAME": "",
-	"usage: minikube addons disable ADDON_NAME": "",
-	"usage: minikube addons enable ADDON_NAME": "",
-	"usage: minikube addons images ADDON_NAME": "",
-	"usage: minikube addons list": "",
-	"usage: minikube addons open ADDON_NAME": "",
-	"usage: minikube config unset PROPERTY_NAME": "",
-	"usage: minikube delete": "",
-	"usage: minikube profile [MINIKUBE_PROFILE_NAME]": "",
+	"użycie: minikube addons configure ADDON_NAME": "",
```
[the remark this answers is not in the document]
Comment (JacekDuszenko) [reply translations/pl.json]: Thank you! My replace tool failed here. I've just fixed this section

File: translations/pl.json [hunk 2]
```diff
@@ -23,9 +23,9 @@
 	"--kvm-numa-count range is 1-8": "",
 	"--network flag is only valid with the docker/podman and KVM drivers, it will be ignored": "",
 	"\u003ctarget file absolute path\u003e must be an absolute Path. Relative Path is not allowed (example: \"/home/docker/copied.txt\")": "",
-	"==\u003e Audit \u003c==": "",
-	"==\u003e Last Start \u003c==": "",
-	"A VPN or firewall is interfering with HTTP access to the minikube VM. Alternatively, try a different VM driver: https://minikube.sigs.k8s.io/docs/start/": "",
+	"==\u003e Audit \u003c==": "==> Audyt <==",
```
Comment (spowelljr) [inline translations/pl.json:26]: Lets leave the symbols as escaped unicode to ensure no inconsistencies
`==\u003e Audyt \u003c==`

File: translations/pl.json [hunk 3]
```diff
@@ -23,9 +23,9 @@
...
+	"==\u003e Last Start \u003c==": "==> Ostatni start <==",
```
Comment (spowelljr) [inline translations/pl.json:27]: Same here as well

File: translations/pl.json [hunk 4]
```diff
@@ -36,57 +36,57 @@
...
-	"Amount of RAM allocated to the minikube VM (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g)": "Ilość zarezerwowanej pamięci RAM dla maszyny wirtualnej minikube (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or )",
-	"Amount of RAM allocated to the minikube VM (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g).": "Ilość zarezerwowanej pamięci RAM dla maszyny wirtualnej minikube (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or )",
+	"Amount of RAM allocated to the minikube VM (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g)": "Ilość zarezerwowanej pamięci RAM dla maszyny wirtualnej minikube (format: \u003cnumber\u003e[\u003cunit\u003e], gdzie jednostka to = b, k, m lub g)",
+	"Amount of RAM allocated to the minikube VM (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g).": "Ilość zarezerwowanej pamięci RAM dla maszyny wirtualnej minikube (format: \u003cnumber\u003e[\u003cunit\u003e], gdzie jednostka to = b, k, m or g)",
```
Comment (spowelljr) [inline translations/pl.json:57]: This line is a duplicate of the line above and no longer exists in minikube, please delete it

File: translations/pl.json [hunk 5]
```diff
@@ -36,57 +36,57 @@
...
+	"At least needs control plane nodes to enable addon": "Wymaga węzłów z płaszczyzny kontrolnej do włączenia addona",
+	"Automatically selected the {{.driver}} driver": "Automatycznie wybrano sterownik {{.driver}}",
+	"Automatically selected the {{.driver}} driver. Other choices: {{.alternates}}": "Automatycznie wybrano sterownik {{.driver}}. Inne możliwe sterowniki: {{.alternates}}",
 	"Available Commands": "Dostępne polecenia",
 	"Basic Commands:": "Podstawowe polecenia",
-	"Because you are using a Docker driver on {{.operating_system}}, the terminal needs to be open to run it.": "",
+	"Because you are using a Docker driver on {{.operating_system}}, the terminal needs to be open to run it.": "Z powodu użycia sterownika dockera na systemie operacyjnym {{.operating_system}}, terminal musi zostać uruchomiony.",
 	"Bind Address: {{.Address}}": "",
-	"Booting up control plane ...": "",
+	"Booting up control plane ...": "Uruchamianie płaszczyzny kontrolnej",
```
Comment (spowelljr) [inline translations/pl.json:71]: Add the `...` at the end to indicate it's in-progress

File: translations/pl.json [hunk 6]
```diff
@@ -356,109 +356,109 @@
 	"Kubernetes {{.new}} is now available. If you would like to upgrade, specify: --kubernetes-version={{.prefix}}{{.new}}": "",
 	"Kubernetes {{.version}} is not supported by this release of minikube": "",
 	"Launching Kubernetes ...": "Uruchamianie Kubernetesa...",
-	"Launching proxy ...": "",
+	"Launching proxy ...": "Uruchamianie proxy",
```
Comment (spowelljr) [inline translations/pl.json:359]: Add the `...` at the end to indicate it's in-progress

File: translations/pl.json [hunk 7]
```diff
@@ -356,109 +356,109 @@
...
-	"Lists all available minikube addons as well as their current statuses (enabled/disabled)": "",
+	"Listening to {{.listenAddr}}. This is not recommended and can cause a security vulnerability. Use at your own risk": "Nasłuchiwanie na adresie {{.listenAddr}}. Jest to niezalecane i może spowodować powstanie podaności bezpieczeństwa. Używaj na własne ryzyko",
+	"Lists all available minikube addons as well as their current statuses (enabled/disabled)": "Wylistuj wszystkie dostępne addony minikube razem z ich obecnymi statusami (włączony/wyłączony)",
 	"Lists all minikube profiles.": "Wylistuj wszystkie profile minikube",
-	"Lists all valid default values for PROPERTY_NAME": "",
-	"Lists all valid minikube profiles and detects all possible invalid profiles.": "",
-	"Lists the URLs for the services in your local cluster": "",
-	"Load a image into minikube": "",
-	"Local folders to share with Guest via NFS mounts (hyperkit driver only)": "",
+	"Lists all valid default values for PROPERTY_NAME": "Wylistuj wszystkie prawidłowe domyślne wartości dla opcji konfiguracyjnej NAZWA_OPCJI",
```
Comment (spowelljr) [inline translations/pl.json:371]: `PROPERTY_NAME` is a key and should be left untranslated

File: translations/pl.json [hunk 8]
```diff
@@ -356,109 +356,109 @@
...
+	"More information: https://docs.docker.com/engine/install/linux-postinstall/#your-kernel-does-not-support-cgroup-swap-limit-capabilities": "Więcej informacji: https://docs.docker.com/engine/install/linux-postinstall/#your-kernel-does-not-support-cgroup-swap-limit-capabilities",
+	"Most users should use the newer 'docker' driver instead, which does not require root!": "Większość użytkowników powinna używać nowszego sterownika docker, ktory nie wymaga uruchamiania z poziomu roota!",
 	"Mount type:   {{.name}}": "",
 	"Mounting host path {{.sourcePath}} into VM as {{.destinationPath}} ...": "",
 	"Mounts the specified directory into minikube": "Montuje podany katalog wewnątrz minikube",
 	"Mounts the specified directory into minikube.": "Montuje podany katalog wewnątrz minikube",
-	"Multiple errors deleting profiles": "",
-	"Multiple minikube profiles were found - ": "",
+	"Multiple errors deleting profiles": "Wystąpiło wiele błędów podczas usuwania profili",
+	"Multiple minikube profiles were found - ": "Znaleziono wiele profili minikube",
```
Comment (spowelljr) [inline translations/pl.json:394]: Add ` - `, ensuring a space at the end of the line as a list is appended

File: translations/pl.json [hunk 9]
```diff
@@ -356,109 +356,109 @@
...
+	"No IP address provided. Try specifying --ssh-ip-address, or see https://minikube.sigs.k8s.io/docs/drivers/ssh/": "Nie znaleziono adresu IP. Spróbuj przekazać adres IP za pomocą flagi --ssh-ip-address lub odwiedź https://minikube.sigs.k8s.io/docs/drivers/ssh/",
+	"No changes required for the \"{{.context}}\" context": "Żadne zmiany nie są wymagane dla kontekstu \"{{.context}}\"",
+	"No minikube profile was found. ": "Nie znaleziono żadnego profilu minikube",
+	"No possible driver was detected. Try specifying --driver, or see https://minikube.sigs.k8s.io/docs/start/": "Nie znaleziono żadnego możliwego sterownika. Spróbuj przekazać sterownik za pomocą flagi --driver lub odwiedź https://minikube.sigs.k8s.io/docs/start/",
+	"No such addon {{.name}}": "Nie istnieje addon {{.name}}",
+	"Node {{.name}} failed to start, deleting and trying again.": "Węzeł {{.name}} nie uruchomił się pomyślnie. Usuwam i próbuję uruchomić węzeł ponownie",
+	"Node {{.name}} was successfully deleted.": "Węzeł {{.name}} został pomyślnie usunięty",
+	"Node {{.nodeName}} does not exist.": "Węzeł {{.name}} nie istnieje",
```
Comment (spowelljr) [inline translations/pl.json:406]: `nodeName` instead of `name`

File: translations/pl.json [hunk 10]
```diff
@@ -356,109 +356,109 @@
...
 	"Please provide a path or url to build": "",
 	"Please provide an image in your local daemon to load into minikube via \u003cminikube image load IMAGE_NAME\u003e": "",
 	"Please re-eval your docker-env, To ensure your environment variables have updated ports:\n\n\t'minikube -p {{.profile_name}} docker-env'\n\n\t": "",
 	"Please re-eval your podman-env, To ensure your environment variables have updated ports:\n\n\t'minikube -p {{.profile_name}} podman-env'\n\n\t": "",
-	"Please see {{.documentation_url}} for more details": "",
-	"Please specify the directory to be mounted: \n\tminikube mount \u003csource directory\u003e:\u003ctarget directory\u003e   (example: \"/host-home:/vm-home\")": "",
+	"Please see {{.documentation_url}} for more details": "Zobacz {{.documentation_url}} żeby uzyskać więcej informacji",
+	"Please specify the directory to be mounted: \n\tminikube mount \u003csource directory\u003e:\u003ctarget directory\u003e   (example: \"/host-home:/vm-home\")": "Sprecyzuj katalog, który ma być zamontowany: \n\tminikube mount <katalog źródłowy>:<katalog docelowy>   (przykład: \"/host-home:/vm-home\")",
```
Comment (spowelljr) [inline translations/pl.json:451]: Leave unicode here as well

File: translations/pl.json [hunk 11]
```diff
@@ -947,20 +947,20 @@
 	"{{.driver_name}} couldn't proceed because {{.driver_name}} service is not healthy.": "",
 	"{{.driver_name}} has less than 2 CPUs available, but Kubernetes requires at least 2 to be available": "",
 	"{{.driver_name}} has only {{.container_limit}}MB memory but you specified {{.specified_memory}}MB": "",
-	"{{.driver}} only has {{.size}}MiB available, less than the required {{.req}}MiB for Kubernetes": "",
-	"{{.extra_option_component_name}}.{{.key}}={{.value}}": "",
+	"{{.driver}} only has {{.size}}MiB available, less than the required {{.req}}MiB for Kubernetes": "sterownik {{.driver}} ma tylko {{.size}}MiB dostępnej przestrzeni dyskowej, to mniej niż wymagane {{.req}}MiB dla Kubernetesa",
+	"{{.extra_option_component_name}}.{{.key}}={{.value}}": "{{.extra_option_component_name}}.{{.key}}={{.value}}",
```
Comment (spowelljr) [inline translations/pl.json:951]: If there's no difference between the translated version can be left empty, but doesn't matter either way

**Review by spowelljr — CHANGES_REQUESTED** [review]

**Comment by JacekDuszenko** [comment]
Thank you for the review Steven, I addressed your comments in the current commit. 
@spowelljr

**Review by spowelljr — APPROVED** [review]

**Review by spowelljr — APPROVED** [review]

**Pull request merged**